redash/query_runner/clickhouse.py

```diff
@@ -112,9 +101,10 @@ def run_query(self, query, user):
             q = self._clickhouse_query(query)
             data = json.dumps(q, cls=JSONEncoder)
             error = None
-        except Exception as exc:
+        except Exception as e:
             data = None
-            error = str(exc)
+            logging.exception(e)
+            error = str(e)
```

Comment:
Btw, it's possible the exception might have some unicode characters, no? (like fragments of the query) So maybe better to use `unicode` here.